### ISSUE TITLE
Images and labels chosen seem to be random while running on Linux

### ISSUE THREAD

**Issue by v-dvorak** [v-dvorak/omr-layout-analysis#1]
The current way of choosing only a part of a dataset is not suited for running on different OS than W11. The program selects a correct number of items but does not link them together correctly.

![image](https://github.com/v-dvorak/omr-layout-analysis/assets/98742752/1cedb83c-6cf6-4a4a-b9d2-fc06550537b8)

## To reproduce:

run on Linux

```
python3 omr-layout-analysis/audiolabs_parser.py file_name -c 20 --al2
```

## Possible fix:

Check all images and all labels in a directory and choose from them only after linking them together.

**Comment by v-dvorak** [comment]
Fixed with c3470ee8ae011718516eb4a54543572687caf55f

**Comment by v-dvorak** [comment]
Fixed with c3470ee8ae011718516eb4a54543572687caf55f
